.github/workflows/links.yml

```diff
@@ -38,3 +41,12 @@ jobs:
           title: Link Checker Report
           content-filepath: ./lychee/out.md
           labels: report, automated issue
+
+      - name: Notify in Slack in case of failure
+        id: slack-notification
+        if: ${{ github.event_name == 'schedule' && env.lychee_exit_code != 0 }}
```

Comment:
as on other PRs, lychee code instead of just failure()?

Reply:
yeah but could the condition not also just be `failure()`?
I think this message was meant on why the issue is created and I'm fine with that.

Reply:
![image](https://github.com/camunda/camunda-tf-eks-module/assets/9141483/34acfdbd-603e-4b57-807c-34dad3fc1efe)

Reply:
now fixed